frontend/packages/dev-console/integration-tests/features/addFlow/create-from-git.feature

```diff
@@ -246,6 +246,18 @@ Feature: Create Application from git form
               And user is able to see Redirect value is selected in "Insecure traffic"
 
 
+        @regression @odc-7167
+        Scenario: Create Serverless Function from Import From Git Form on Add page: A-06-TC19
+            Given user has installed OpenShift Serverless Operator
+              And user has selected namespace "aut-addflow-git"
+              And user clicks on the Import from Git card on the Add page
+             When user enters Git Repo URL as "https://github.com/Lucifergene/oc-func"
+              And user clicks Create button on Add page
```

Comment:
Can you also add steps to add the name "oc-func" before creating the application